integrationTests/testSyncNode.go

```diff
@@ -81,6 +81,8 @@ func NewTestSyncNode(
 		})
 
 	messenger := CreateMessengerWithNoDiscoveryAndPeersRatingHandler(peersRatingHandler)
+	genericEpochNotifier := forking.NewGenericEpochNotifier()
```

Comment:
We should use the same epoch notifier for all components, instead of creating a new one for each subcomp.

E.g.: Here you create a new epoch notifier to be used by `enableEpochsHandler` and another one which is used for `TestNodeProcessor`, `L105`

Reply:
removed. These were tentative fixes for integration tests.